hack/.ci/run-e2e-gke.sh

```diff
@@ -60,6 +60,46 @@ kubectl_create -f - <<EOF
     - patch
 EOF
 
+# FIXME: remove the workaround once https://github.com/scylladb/scylla-operator/issues/749 is done
+kubectl_create -n default -f - <<EOF
+  apiVersion: apps/v1
+  kind: DaemonSet
+  metadata:
+    name: sysctl
+  spec:
+    selector:
+      matchLabels:
+        app.kubernetes.io/name: sysctl
+    template:
+      metadata:
+        labels:
+          app.kubernetes.io/name: sysctl
+      spec:
+        containers:
+        - name: sysctl
+          securityContext:
+            privileged: true
+          image: "${SO_IMAGE}"
+          imagePullPolicy: IfNotPresent
+          command:
+          - /usr/bin/bash
+          - -euExo
+          - pipefail
+          - -O
+          - inherit_errexit
+          - -c
+          args:
+          - |
+            sysctl fs.aio-max-nr=0xffffffff
+
+            sleep infinity &
+            wait
```

Comment:
how is this different from `sleep infinity` on the foreground

Reply:
sigint handling?

Reply:
interesting, sending `SIGINT` to `sleep` works for me
```
 tnozicka  ~  sleep infinity
^C
 tnozicka  ~  SIGINT  
```

Reply:
Doesn't ^C send a SIGINT to the entire process group?
`kill -s INT -$PGID` as opposed to `kill -s INT $PID`

Reply:
Shouldn't you kill the bash process in that case?

```bash
~ cat script.sh
#!/bin/bash
sleep infinity
~ ./script.sh &
~ ps o pid,ppid,pgid,comm
   6467    6289    6467 zsh
  13292    6467   13292 script.sh
  13294   13292   13292 sleep
  13500    6467   13500 ps
~ kill -s INT 13292
~ ps o pid,ppid,pgid,comm
   6467    6289    6467 zsh
  13292    6467   13292 script.sh
  13294   13292   13292 sleep
  13931    6467   13931 ps
~ kill -s INT -13292
[1]  + 13292 interrupt  ./script.sh
```